interactions/api/models/audit_log.py

```diff
@@ -0,0 +1,277 @@
+from enum import IntEnum
+from typing import TYPE_CHECKING, List, Optional, TypeVar
+
+from .attrs_utils import DictSerializerMixin, convert_list, define, field
+from .channel import Channel
+from .misc import Snowflake
+from .user import User
+from .webhook import Webhook
+
+__all__ = (
+    "AuditLogEntry",
+    "AuditLogEvents",
+    "AuditLogs",
+    "AuditLogChange",
+    "OptionalAuditEntryInfo",
+)
+
+_T = TypeVar("_T")
+
+
+if TYPE_CHECKING:
+    from .guild import Integration, ScheduledEvents
+    from .gw import AutoModerationRule
+
+
+class AuditLogEvents(IntEnum):
+    """
+    A class object representing the different types of AuditLogEvents.
+
+    :ivar int GUILD_UPDATE: 1 - Server settings were updated
+    :ivar int CHANNEL_CREATE: 10 - Channel was created
+    :ivar int CHANNEL_UPDATE: 11 - Channel settings were updated
+    :ivar int CHANNEL_DELETE: 12 - Channel was deleted
+    :ivar int CHANNEL_OVERWRITE_CREATE: 13 - Permission overwrite was added to a channel
+    :ivar int CHANNEL_OVERWRITE_UPDATE: 14 - Permission overwrite was updated for a channel
+    :ivar int CHANNEL_OVERWRITE_DELETE: 15 - Permission overwrite was deleted from a channel
+    :ivar int MEMBER_KICK: 20 - Member was removed from server
+    :ivar int MEMBER_PRUNE: 21 - Members were pruned from server
+    :ivar int MEMBER_BAN_ADD: 22 - Member was banned from server
+    :ivar int MEMBER_BAN_REMOVE: 23 - Server ban was lifted for a member
+    :ivar int MEMBER_UPDATE: 24 - Member was updated in server
+    :ivar int MEMBER_ROLE_UPDATE: 25 - Member was added or removed from a role
+    :ivar int MEMBER_MOVE: 26 - Member was moved to a different voice channel
+    :ivar int MEMBER_DISCONNECT: 27 - Member was disconnected from a voice channel
+    :ivar int BOT_ADD: 28 - Bot user was added to server
+    :ivar int ROLE_CREATE: 30 - Role was created
+    :ivar int ROLE_UPDATE: 31 - Role was updated
+    :ivar int ROLE_DELETE: 32 - Role was deleted
+    :ivar int INVITE_CREATE: 40 - Server invite was created
+    :ivar int INVITE_UPDATE: 41 - Server invite was updated
+    :ivar int INVITE_DELETE: 42 - Server invite was deleted
+    :ivar int WEBHOOK_CREATE: 50 - Webhook was created
+    :ivar int WEBHOOK_UPDATE: 51 - Webhook properties or channel were updated
+    :ivar int WEBHOOK_DELETE: 52 - Webhook was deleted
+    :ivar int EMOJI_CREATE: 60 - Emoji was created
+    :ivar int EMOJI_UPDATE: 61 - Emoji name was updated
+    :ivar int EMOJI_DELETE: 62 - Emoji was deleted
+    :ivar int MESSAGE_DELETE: 72 - Single message was deleted
+    :ivar int MESSAGE_BULK_DELETE: 73 - Multiple messages were deleted
+    :ivar int MESSAGE_PIN: 74 - Message was pinned to a channel
```

Comment:
```suggestion
    :ivar int MESSAGE_PIN: 74 - Message was pinned in a channel
```
You don't pin a message to a channel. You pin a message that is in a channel.
Resolve if you just copied from discord docs